pep-0677.rst

```diff
@@ -27,116 +27,127 @@ work as a drop-in replacement.
 Motivation
 ==========
 
-Describing Callable Signatures with ``typing.Callable``
--------------------------------------------------------
-
 One way to make code safer and easier to analyze is by making sure
 that functions and classes are well-typed. In Python we have type
 annotations defined by PEP 484 to provide type hints that can find
```

Comment:
This wording still sounds a little strange, to my ears. "Type annotations" are not defined by PEP 484; PEP 484 defines the _framework_ for type annotations. I would revise it to either:

```
In Python we have type annotations, first introduced by PEP 484,
```

or

```
In Python we have type annotations, the framework for which is defined in PEP 484,
```